tests/acceptance/api_test.go

```diff
@@ -1148,7 +1141,7 @@ func TestApi(t *testing.T) {
 			// with invalid adminRoleArn
 			res := apiRequest(t, &apiRequestInput{
 				method: "PUT",
-				url:    apiURL + "/accounts/not-an-account-id",
+				url:    apiURL + "/accounts/123456789012",
```

Comment:
accountIDs will have to be 12 digit #s going forward.  This will fail validation before it will even try to get an account.